docs/_docs/snapshots/snapshots.adoc

```diff
@@ -178,9 +172,59 @@ the topology and wait while the data gets rebalanced and indexes are rebuilt.
 |N > M | Unsupported.
 |===
 
+=== Automatic snapshot restore procedure
+
+The automatic restore procedure allows the user to restore cache groups from a snapshot on an active cluster using the Java API or link:tools/control-script[command line script].
+
+Currently, this procedure has a number of limitations:
+
+* Restoring is possible only if all parts of the snapshot are present in the cluster. Each node looks a local data snapshot in the configured snapshot path by the given snapshot name and consistent node ID.
```

Comment:
Each node looks a local data snapshot > Maybe add for? I am not sure what this sentence is supposed to mean.
consistent node ID > do we need to write about this in docs? I assume it happens on all nodes.

Each node looks for a local data snapshot in the configured snapshot path. It uses the provided snapshot name.

Reply:
Thanks, I changed it to
``Each node looks for a local snapshot data in the configured snapshot path by the given snapshot name and consistent node ID.``

The main purpose of this sentence is to inform the user that the snapshot data is located under ``../{snapshotName}/../{consistentId}/..``, so if the consistent ID of the node is changed - the procedure will not be able to find snapshot data.

Reply:
LGTM!